### PR TITLE
[hpx] Update to V1.4.1

### PR DESCRIPTION
**Pull request by hkaiser**

### CODE REVIEW

File: ports/hpx/portfile.cmake [hunk 1]
```diff
@@ -4,8 +4,8 @@ vcpkg_check_linkage(ONLY_DYNAMIC_LIBRARY)
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO STEllAR-GROUP/hpx
-    REF 1.4.0
-    SHA512 501dbea384f8daa725f752f78198ba8af7b41bf0da432655fb0eaf588ccb1cec40528e28c6dc898230090d04e3dd2c50f7cadd753a13d15b51cea0ac881c3ffd
+    REF 1.4.1
+    SHA512 f110d5e5c10ec396f6e762568c9ecd5b767cb6efe91168b5caa8fe1e07bb5870cd13b3392fa4e008a2cc0e044b02084a35b0866e943d9b9c7435599c131f1582
     HEAD_REF stable
 )
```
Comment (NancyLi1013) [inline ports/hpx/portfile.cmake]: Could you please use `vcpkg_replace_string() `to replace a string in a file?

Comment (NancyLi1013) [reply ports/hpx/portfile.cmake]: Could you please update
```
file(INSTALL
    ${SOURCE_PATH}/LICENSE_1_0.txt
    DESTINATION ${CURRENT_PACKAGES_DIR}/share/hpx RENAME copyright)
```
as
```
file(INSTALL
    ${SOURCE_PATH}/LICENSE_1_0.txt
    DESTINATION ${CURRENT_PACKAGES_DIR}/share/${PORT} RENAME copyright)
```
?

File: ports/hpx/CONTROL [hunk 2]
```diff
@@ -1,5 +1,5 @@
 Source: hpx
-Version: 1.4.0-1
+Version: 1.4.1-1
```
Comment (NancyLi1013) [inline ports/hpx/CONTROL]: The `Version `should be `1.4.1` here.

**Review by NancyLi1013 — CHANGES_REQUESTED** [review]

**Review by NancyLi1013 — APPROVED** [review]

**Comment by strega-nil** [comment]
Thanks @hkaiser !

**Pull request merged**